getData.js

```diff
@@ -0,0 +1,33 @@
+module.exports = ({context, core}) => {
```

Comment:
is it possible for this to be a python script? is there a reason why JS was chosen?

Reply:
can this file also be located in `.github/workflows`